openai/openai_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
+	"time"
 
 	"github.com/gin-gonic/gin"
```

Comment:
Can we remove Gin from this test file (in the PR this branches off of)

Reply:
Gin is needed to test the middleware, do you mind explaining what you mean?